HardCodedStringCheckerSharp.UnitTests/HardCodedStringCheckerSharp.UnitTests.csproj

```diff
@@ -69,6 +69,7 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
+    <Compile Include="AppControllerTest.cs" />
```

Comment:
Presumably it's missing the declaration of the AppControllerTest class?

Reply:
Oh, shoot, I goofed the patch.